packages/devtools/src/constants.js

```diff
@@ -23,17 +23,20 @@ export const DEFAULT_FLAGS = [
     '--mute-audio',
     // Skip first run wizards
     '--no-first-run',
+    '--no-default-browser-check',
     // chromedriver flags
     '--disable-hang-monitor',
     '--disable-prompt-on-repost',
     '--disable-client-side-phishing-detection',
+    '--disable-features=TranslateUI',
```

Comment:
Should be removed as UItranslation is already disabled [here](https://github.com/koggdal/webdriverio/blob/devtools-fix-ignore-chrome-flags/packages/devtools/src/constants.js#L45)

Reply:
Oh good catch! Thanks!